components/repairshopr/sources/common/source.mjs

```diff
@@ -0,0 +1,74 @@
+import app from "../../repairshopr.app.mjs";
+
+export default {
+  dedupe: "unique",
+  props: {
+    app,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
```

Comment:
Can we just use `DEFAULT_POLLING_SOURCE_TIMER_INTERVAL` from `@pipedream/platform` like it is done [here](https://github.com/PipedreamHQ/pipedream/blob/f8ff2ffb27265ef154c1f69c08440d70f9bc4fec/components/podio/sources/common/common-poll.mjs#L11)